packages/server-wallet/src/protocols/__test__/challenge-submitter.test.ts

```diff
@@ -51,11 +51,9 @@ describe(`challenge-submitter`, () => {
       data: {targetChannelId: c.channelId},
     };
 
-    const dbObjective = await knex.transaction(async tx =>
-      ObjectiveModel.insert<DBSubmitChallengeObjective>(obj, false, tx)
-    );
+    const wO = await knex.transaction(async tx => ObjectiveModel.insert(obj, false, tx));
```

Comment:
Could we use a more descriptive variable name?

Reply:
8beb219